DigitalLearningSolutions.Web/Views/SuperAdmin/Users/UserCentreAccounts.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.UserCentreAccounts
+@using DigitalLearningSolutions.Web.ViewComponents
+@model UserCentreAccountRoleViewModel
+@{
+    ViewData["Title"] = "UserCentreAccounts";
```

Comment:
There should be spaces between the words in the title "User Centre Accounts"

Reply:
Review comments implemented and checked in. please review.